api-client/src/mapping/mapCategoryJson.ts

```diff
@@ -3,18 +3,14 @@ import CategoryModel from '../models/CategoryModel'
 import moment from 'moment-timezone'
 import mapAvailableLanguages from '../mapAvailableLanguages'
 import normalizePath from '../normalizePath'
-import sanitizeHtml from 'sanitize-html-react'
+import { sanitize } from 'isomorphic-dompurify'
 
 const mapCategoryJson = (json: JsonCategoryType, basePath: string): CategoryModel =>
   new CategoryModel({
     root: false,
     path: normalizePath(json.path),
     title: json.title,
-    content: sanitizeHtml(json.content, {
-      allowedSchemes: ['http', 'https', 'data', 'tel', 'mailto'],
-      allowedTags: false,
-      allowedAttributes: false
-    }),
+    content: sanitize(json.content),
```

Comment:
was there a reason why we didn't support ftp before?

Reply:
@maxammann

Reply:
I dont think so, just no usecase. FTP is also not supported in most browsers afaik.